models/DataObject/ClassDefinition/Data/Multiselect.php

```diff
@@ -359,6 +359,19 @@ public function getDataFromEditmode($data, $object = null, $params = [])
         return $data;
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getDiffDataFromEditmode($data, $object = null, $params = []): ?array
+    {
+        $data = $data[0]['data'];
+        if (is_string($data) && strlen($data)) {
```

Comment:
```suggestion
        if (is_string($data) && $data !== '') {
```
$data !== '' is slightly faster than strlen and is in my opion more obvious